packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../global/functions';
+
+.usa-label {
+  font-size: rem-override(1.06rem);
+}
+
+.usa-input {
```

Comment:
Do we really want all of these styles to be loaded into every single component? I'm assuming the approach is to import `@import '../../global/formation_overrides';` into all USWDS v3 component variations? But a component like banner would not need these styles.

Reply:
In this case, even though the file is located within the 'global' folder, it is not automatically imported globally (same for the _functions.scss file). What causes the file to be imported is on [line 8](https://github.com/department-of-veterans-affairs/component-library/pull/622/files#diff-3e956d46d7983277911085179f5eb079e722c72953af3196d26cce81399364dfR8) of the va-text-input.scss file. So, the idea is that we would import the formation_overrides file only where we need it; it's in the 'global' folder only as a signifier that it is 'available' globally, not 'imported' globally. Very good question, thanks for double-checking the intentions!

Reply:
What I mean is that is that all of the CSS will be loaded in components that don't necessarily need those styles (not that it's global to the site).

Reply:
Agreed that this is a less-than-ideal solution long term, but an acceptable solution for now. Will revisit if and when this overrides file gets "too big" to afford to import everywhere it's needed.

Reply:
FYI I'm marking this unresolved not because I think there needs more work in this PR but so that the conversation is easier to reference in the future if we need to come back and reorganize the stylesheet. Thanks for clarifying this! 👍